src/App.css

```diff
@@ -211,7 +200,7 @@ a.text-link {
   box-sizing: border-box;
 }
 
-.subthought-text, .thought-annotation .breadcrumbs {
```

Comment:
This is still needed. The thought annotation is a clone of a thought used to render superscripts in the correct position without making them editable (as they would be in the regular thought element).

Reply:
For some reason this line was unexpected deleted. I did not made this change at all.

Reply:
Same here, the deletion is still part of this PR. You will need to revert it to the original.